### PR TITLE
chore(pageserver): temporary metrics on ingestion time

### PR DESCRIPTION
**Pull request by skyzh**
## Problem

As a follow-up on https://github.com/neondatabase/neon/pull/7467, also measure the ingestion operation speed.

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics? if so did you add the relevant metrics to the dashboard?
- [ ] If this PR requires public announcement, mark it with /release-notes label and add several sentences in this section.

## Checklist before merging

- [ ] Do not forget to reformat commit message to not include the above checklist

### CODE REVIEW

File: pageserver/src/metrics.rs [hunk 1]
```diff
@@ -1749,6 +1720,43 @@ macro_rules! redo_bytes_histogram_count_buckets {
...
+    .expect("failed to define a metric"),
+    records_filtered: register_int_counter!(
+        "pageserver_wal_ingest_records_filtered",
+        "Number of WAL records filtered out due to sharding"
+    )
+    .expect("failed to define a metric"),
+    time_spent_on_ingest: register_histogram!(
+        "pageserver_wal_ingest_put_value_seconds",
+        "Actual time spent on ingesting a record",
+        redo_histogram_time_buckets!(),
```
Comment (skyzh) [inline pageserver/src/metrics.rs:1755]: moved the definition further down due to `redo_histogram_time_buckets` is defined after the original place.

File: pageserver/src/pgdatadir_mapping.rs [hunk 2]
```diff
@@ -1551,6 +1552,8 @@ impl<'a> DatadirModification<'a> {
     pub async fn commit(&mut self, ctx: &RequestContext) -> anyhow::Result<()> {
         let mut writer = self.tline.writer().await;
 
+        let timer = WAL_INGEST.time_spent_on_ingest.start_timer();
```
Comment (VladLazar) [inline pageserver/src/pgdatadir_mapping.rs:1555]: Technically this is a hot code path, but it's probably fine. Can you create a follow up to remove this when you're done with it?

Comment (skyzh) [reply pageserver/src/pgdatadir_mapping.rs:1555]: yep I plan to remove it two weeks later

Comment (VladLazar) [reply pageserver/src/pgdatadir_mapping.rs:1555]: Let's open an issue for it. I always forget to come back to this sort of stuff :smile:

**Pull request merged**